lib/retry.js

```diff
@@ -32,7 +26,12 @@ inherits(MaxRetryError, Error);
 // it will retry by running the `promiser` function again. Retry will
 // stop when it has tried `maxRetries` times or if the promise fails
 // with the special error `ERR_ENDRETRY`.
-export default function retry(promiser) {
+export default function retry(promiser, _options) {
+  const options = Object.assign({
+    maxRetries: 3,
+    timeFunction: i => 100 * Math.pow(i, 2),
```

Comment:
We were wanting to adjust the time function so it spreads the retries over more time. Currently the delays for 3 retries is around 2 seconds, and we would want it to take minimum of 30 seconds.

Reply:
This does not apply to the WebSocket connection as we don't use `retry` there, only for clock and axios requests, would that still make sense?

Reply:
This would apply only for clock and axios requests as WS does not use it, would `30000 * Math.pow(i, 2)` make sense?

Reply:
I think 30,000 is too high - there would  be a 2 minute delay for the second retry.
This doesn't need to apply to WS since it doesn't have a limited number of retries.